### PR TITLE
fix: stopTimeout

### PR DESCRIPTION
**Pull request by Faf4a**

### CODE REVIEW

File: src/functions/stopTimeout.js [hunk 1]
```diff
@@ -3,16 +3,20 @@ module.exports = async (d) => {
...
     const [id] = data.inside.splits;
+    const MAX_SAFE_INTEGER = 0x7FFFFFFF;
 
     const timeout = await d.client.db.get(
         "__aoijs_vars__",
         "setTimeout",
         id,
     );
-    if (!timeout) return d.error(`Timeout with ID: ${id} not found.`);
+    if (!timeout) return d.error(`AoiError: Timeout ID ${id} not found.`);
```
Comment (Leref) [inline src/functions/stopTimeout.js]: Refer this actual using d.aoiError

File: src/functions/stopTimeout.js [hunk 2]
```diff
@@ -3,16 +3,20 @@ module.exports = async (d) => {
     if (data.err) return d.error(data.err);
 
     const [id] = data.inside.splits;
+    const MAX_SAFE_INTEGER = 0x7FFFFFFF;
```
Comment (Leref) [inline src/functions/stopTimeout.js]: @USERSATOSHI ?

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/functions/stopTimeout.js (hunk 2 above)
Comment (Faf4a) [reply src/functions/stopTimeout.js:6]: thats the max safe integer which prevents the timeout from overflowing

Comment (USERSATOSHI) [reply src/functions/stopTimeout.js]: > thats the max safe integer which prevents the timeout from overflowing

better use Number.MAX_SAFE_INTEGER ig?

Comment (USERSATOSHI) [reply src/functions/stopTimeout.js]: nvm that wont work , but better have different name, something like MAX_TIMEOUT_DURATION or something

Comment (Faf4a) [reply src/functions/stopTimeout.js:6]: yop

**Review by Leref — APPROVED** [review]
Looks good.

**Pull request merged**